test/e2e/tests/comments/inline.spec.ts

```diff
@@ -16,96 +16,173 @@ interface InlineCommentCreationTestProps {
-  await page.waitForSelector('[data-testid="comment-input"]', {
+  await page.getByTestId('comment-input').waitFor({
     ...WAIT_OPTIONS,
     state: 'visible',
   })
-  const commentInput = page.locator('[data-testid="comment-input"]')
-  await expect(commentInput).toBeVisible()
+  const getCommentInput = () => page.locator('[data-testid="comment-input"]')
+  await expect(getCommentInput()).toBeVisible()
 
-  await page.waitForSelector('[data-inline-comment-state="authoring"]', WAIT_OPTIONS)
-  const authoringDecorator = page.locator('[data-inline-comment-state="authoring"]')
-  await expect(authoringDecorator).toHaveText(PTE_CONTENT_TEXT)
+  const authoringDecorator = '[data-inline-comment-state="authoring"]'
+  await page.locator(authoringDecorator).waitFor(WAIT_OPTIONS)
+  await expect(page.locator(authoringDecorator)).toHaveText(PTE_CONTENT_TEXT)
 
   // 6. Author the comment and submit it by clicking the send button.
-  const commentInputTextBox = commentInput.locator('div[role="textbox"]')
-  await commentInputTextBox.fill('This is a comment')
-  const sendButton = commentInput.locator('[data-testid="comment-input-send-button"]')
-  await sendButton.click({
+  await getCommentInput().locator('div[role="textbox"]').fill('This is a comment')
+  await getCommentInput().locator('[data-testid="comment-input-send-button"]').click({
     delay: 1000,
   })
 
   // 7. Verify the comment has been successfully added by checking for the presence of the
   //    comment decorator with the correct content.
-  await page.waitForSelector('[data-inline-comment-state="added"]', WAIT_OPTIONS)
-  const addedDecorator = page.locator('[data-inline-comment-state="added"]')
-  await expect(addedDecorator).toHaveText(PTE_CONTENT_TEXT)
+  const addedDecorator = '[data-inline-comment-state="added"]'
+  await page.locator(addedDecorator).waitFor(WAIT_OPTIONS)
+  await expect(page.locator(addedDecorator)).toHaveText(PTE_CONTENT_TEXT)
 
   // 8. Verify that the comments list is visible after the comment has been added.
-  await page.waitForSelector('[data-testid="comments-list"]', WAIT_OPTIONS)
-  const commentsList = page.locator('[data-testid="comments-list"]')
-  await expect(commentsList).toBeVisible()
+  await page.getByTestId('comments-list').waitFor(WAIT_OPTIONS)
+  await expect(page.locator('[data-testid="comments-list"]')).toBeVisible()
 
   // 9. Verify that the comment appears within the list and correctly references the intended content.
-  await page.waitForSelector('[data-testid="comments-list-item"]', WAIT_OPTIONS)
-  const commentsListItem = page.locator('[data-testid="comments-list-item"]')
-  await expect(commentsListItem).toBeVisible()
-  const commentsListItemReferencedValue = page.locator(
-    '[data-testid="comments-list-item-referenced-value"]',
+  const commentsListItem = '[data-testid="comments-list-item"]'
+  await page.getByTestId('comments-list-item').waitFor(WAIT_OPTIONS)
```

Comment:
I feel we shoud likely be relying on solely the expect than this? Why waiting for the comment list Item? 
(I'm just wondering if it's needed given the line over at 166)